scipy/stats/_continuous_distns.py

```diff
@@ -6376,10 +6376,87 @@ def _stats(self):
     def _entropy(self):
         return _EULER/2.0 + 1 - 0.5*np.log(2)
 
+    def fit(self, data, *args, **kwds):
+        """
+        Maxmimum likelihood estimation of the `scale` parameter
+        is used in the numerical optimization to determine `loc`.
+
+        Parameters
+        ----------
+        data : array_like
+            Data to use in calculating the maximum likelihood estimate.
+
+        Returns
+        -------
+        loc, scale : float
+            Maximum likelihood estimate for the scale, provided loc.
+
+        Notes:
+        If `floc` is fixed, the maximum likelihood scale may be calculated
+        without numerical optimization. Otherwise, numerical optimization
+        from `.rv_continuous` will be  conducted to determine `loc` and its
+        scale based on MLE.
+        """
+
+        data, floc, fscale = _check_fit_input_parameters(self, data,
+                                                         args, kwds)
+
+        def scale_mle(loc, data):
+            # Source: Statistical Distributions, 3rd Edition. Evans, Hastings,
+            # and Peacock (2000), Page 175
+            return (np.sum((data - loc) ** 2) / (2 * len(data))) ** .5
+
+        """
+        if `floc` is not fixed, provide a `_reduce_func` for `.rv_continuous`
+        to use. '_reduce_func' returns:
+            - the parameter to optimize from `x0`, loc
+            - `func` to minimize, and
+            - `get_tuple` to obtain the dependent parameter, as
+               numerical optimization over `func` only indicates
+               the independent parameter's solution.
+        """
+
+        if floc is not None:
+            return floc, scale_mle(floc, data)
+        if fscale is not None:
+            # `fscale` is fixed, use standard numerical optimization
+            return super(rayleigh_gen,
+                         self).fit(data, *args, **kwds)
+        else:
+            # account for user provided guesses
+            loc, scale = self._fitstart(data)
+            loc = kwds.pop('loc', loc)
+            scale = kwds.pop('scale', scale)
```

Comment:
Is this actually going to be used for anything? If not, consider omitting.

Reply:
I included this because of the discussion on 'plateau' behavior of some log likelihood functions from a while back in case that were to be the case here. Whether changing the initial guesses produces better or different results to the user's preference over `self._fitstart` is something I don't know.

Reply:
I meant `scale` specifically. Does the guess for `scale` affect anything? It is not a decision variable in the numerical optimization, since it is set by the analytical formula, so does the user's guess make any difference?

Reply:
No, it would have no effect. I will remove it and note why only user guesses for `loc` are used.